src/v2/guide/routing.md

```diff
@@ -1,16 +1,16 @@
 ---
-title: Routing
+title: Định tuyến
 type: guide
 order: 501
 ---
 
-## Official Router
+## Thư viện Chính thức
 
-For most Single Page Applications, it's recommended to use the officially-supported [vue-router library](https://github.com/vuejs/vue-router). For more details, see vue-router's [documentation](https://router.vuejs.org/).
+[vue-router](https://github.com/vuejs/vue-router) là thư viện chính thức của [Vue](https://github.com/vuejs) và nó được khuyến nghị sử dụng cho phần lớn các Ứng dụng Đơn Trang (Single Page Applications). Để biết chi tiết, hãy xem [tài liệu chính thức](https://router.vuejs.org/) của vue-router.
 
-## Simple Routing From Scratch
+## Định tuyến Đơn giản
 
-If you only need very simple routing and do not wish to involve a full-featured router library, you can do so by dynamically rendering a page-level component like this:
+Nếu bạn chỉ cần một ứng dụng về định tuyến đơn giản mà không muốn quan tâm đến các chức năng đầy đủ của thư viện [vue-router](https://github.com/vuejs/vue-router), bên dưới là ví dụ đơn giản hữu ích cho bạn:
```

Comment:
Ở đây có mấy vấn đề:

- "very simple routing" không thể dịch là "ứng dụng về định tuyến đơn giản," vì subject là hoàn toàn khác nhau (routing vs. app)
- không cần phải link đến vue-router một lần nữa
- nguyên phần "you can do so by dynamically rendering a page-level component like this" dịch thành "bên dưới là ví dụ đơn giản hữu ích cho bạn" thì quá thoáng, tới độ không còn là dịch nữa mà đã trở thành sáng tác.

Reply:
@phanan "you can do so by dynamically rendering a page-level component like this" đoạn này anh có góp ý thêm gì không, chứ thực sự em cũng không nghĩ đoạn này dịch sao ra cho tiếng Việt nó hợp lý hơn. Chính vì thế nên em đã dịch thoáng hơn như trên đó.

Reply:
Có thể dịch thành "bạn có thể render một component ở cấp trang (page-level) như sau" hoặc gì đó tương tự.